packages/cli/src/ceramic-daemon.ts

```diff
@@ -805,10 +832,20 @@ export class CeramicDaemon {
       return
     }
 
-    const indexedModelStreamIDs = await this.ceramic.admin.getIndexedModels()
-    res.json({
-      models: indexedModelStreamIDs.map((modelStreamID) => modelStreamID.toString()),
-    })
+    const indexedModels = await this.ceramic.admin.getIndexedModelData()
+    const body: Record<string, any> = {
+      models: indexedModels.map((idx) => idx.streamID.toString()),
+    }
+    const outputFormat = req.query.outputFormat?.toString() ?? 'document'
```

Comment:
I had been thinking that the default behavior should be the old behavior - i.e. `onlyStreamIds` - but that if you explicitly ask for the `document` format you *just* get the document structure back, without duplicating the `models` field as an array.

That said, I think Paul's suggestion to just use a different API endpoint is probably actually cleaner